src/Nixfmt/Lexer.hs

```diff
@@ -183,6 +183,7 @@ pushTrivia t = modify (<> t)
 lexeme :: Parser a -> Parser (Ann a)
 lexeme p = do
   lastLeading <- takeTrivia
+  SourcePos{Text.Megaparsec.sourceLine = line} <- getSourcePos
```

Comment:
From [CI](https://github.com/NixOS/nixfmt/actions/runs/10003330283/job/27650058187?pr=224) we can see this in the logs for the last commit:

```
[INFO ] #nixfmt-rfc-style: 35101 files processed in 125.91s
```

And from [CI of the previously merged PR](https://github.com/NixOS/nixfmt/actions/runs/9992970991/job/27619275018) we can see this:

```
[INFO ] #nixfmt-rfc-style: 35096 files processed in 132.35s
```

So definitely no significant change there :)

In the future it would be great to include the time in the Nixpkgs diff commit messages